examples/gallery/plot/datetime-inputs.py

```diff
@@ -0,0 +1,58 @@
+"""
+Datetime inputs
+---------------
+
+The :meth:`pygmt.Figure.plot` method can plot datetime inputs of individual types:
+
+- numpy.datetime64: for available options see https://numpy.org/doc/stable/reference/arrays.datetime.html
+- :class:`pandas.DatetimeIndex`
+- :class:`xarray.DataArray`: datetimes included in a *xarray.DataArray*
+- raw datetime strings in ISO format (e.g. ``["YYYY-MM-DD", "YYYY-MM-DDTHH", "YYYY-MM-DDTHH:MM:SS"]``)
+- Python built-in :class:`datetime.datetime` and :class:`datetime.date`
+
+We can pass datetime inputs based on one of the types listed above directly to the ``x`` and ``y`` options.
```

Comment:
They're called **arguments** in Python.
```suggestion
We can pass datetime inputs based on one of the types listed above directly to the ``x`` and ``y`` arguments.
```